src/modules/project/pages1/projectDashboard/views/promote/AmbassadorPayoutSections.tsx

```diff
@@ -0,0 +1,150 @@
+import {
+  Box,
+  Button,
+  FormControl,
+  FormErrorMessage,
+  Input,
+  InputGroup,
+  InputRightElement,
+  SimpleGrid,
+  Spinner,
+  VStack,
+} from '@chakra-ui/react'
+import { t } from 'i18next'
+import React from 'react'
+import { FieldErrors, SubmitHandler, UseFormRegister } from 'react-hook-form'
+
+import { CardLayout } from '@/shared/components/layouts/CardLayout.tsx'
+import { Body } from '@/shared/components/typography/Body.tsx'
+import { ProjectAmbassadorListQuery } from '@/types/index.ts'
+
+import { TableData, TableWithAccordion } from '../../common/TableWithAccordion.tsx'
+
+const ADD_AMBASSADOR_FORM_ID = 'add-ambassador-form'
+const PAYOUT_RATE_MIN = 0.01
+const PAYOUT_RATE_MAX = 99
+
+type AddAmbassadorFormData = {
+  heroId: string
+  payoutRate: number | string
+}
+
+type ProjectType = NonNullable<ProjectAmbassadorListQuery['projectGet']>
+type AmbassadorsConnectionType = NonNullable<ProjectType['ambassadors']>
+type EdgesArrayType = NonNullable<AmbassadorsConnectionType['edges']>
+type AmbassadorEdge = NonNullable<EdgesArrayType[number]>
+
+interface AmbassadorPayoutsSectionProps {
+  ambassadors: AmbassadorEdge[]
+  ambassadorsLoading: boolean
+  ambassadorTableSchema: TableData<AmbassadorEdge>[]
+  handleAddAmbassadorSubmit: (
+    onSubmit: SubmitHandler<AddAmbassadorFormData>,
+  ) => (e?: React.BaseSyntheticEvent) => Promise<void>
+  onAddAmbassador: SubmitHandler<AddAmbassadorFormData>
+  registerAddAmbassador: UseFormRegister<AddAmbassadorFormData>
+  addAmbassadorErrors: FieldErrors<AddAmbassadorFormData>
+  addAmbassadorLoading: boolean
+}
+
+/** AmbassadorPayoutsSection: Displays the Ambassador Payouts table and add form */
+export const AmbassadorPayoutsSection = ({
+  ambassadors,
+  ambassadorsLoading,
+  ambassadorTableSchema,
+  handleAddAmbassadorSubmit,
+  onAddAmbassador,
+  registerAddAmbassador,
+  addAmbassadorErrors,
+  addAmbassadorLoading,
+}: AmbassadorPayoutsSectionProps) => {
+  return (
+    <CardLayout w="full">
+      <VStack spacing={4} align="start">
```

Comment:
can passin the spacing and align property to the carlayout itself, the extra VStack is not required.